lib/src/main/java/com/github/niqdev/openwebnet/OpenWebNetObservable.java

```diff
@@ -88,9 +88,17 @@ private OpenWebNetObservable() {}
         return context -> {
             try {
                 ByteBuffer buffer = context.getEmptyBuffer();
-                Integer count = context.getClient().read(buffer);
-                String message = new String(buffer.array()).trim();
-                log(String.format("read: %d|%s", count, message));
+                SocketChannel channel = context.getClient();
+                Integer count = 1;
```

Comment:
use the primitive `int` in this case, since is what `client.read(buffer)` returns